academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md

```diff
@@ -22,62 +22,111 @@ To create MAKES index, you will need to format your data into MAKES entity files
 
 ### Define functions to extract MAG data
 
-1. Download `samples/CreateFunctions.usql` from your MAG release to your local drive. <br> From [Azure portal](https://portal.azure.com), go to your MAG Azure Storage account > **Containers > [mag-yyyy-mm-dd] > samples > CreateFunctions.usql > Download**.
+::: moniker range="makes-1.0"
 
-   ![Download CreateFunctions.usql](../graph/media/samples-azure-data-lake-hindex/create-functions-download.png "Download CreateFunctions.usql")
+1. Download `samples/CreateFunctions.usql` from your MAG release to your local drive. 
+
+    From [Azure portal](https://portal.azure.com), go to your MAG Azure Storage account > **Containers > [mag-yyyy-mm-dd] > samples > CreateFunctions.usql > Download**.
```

Comment:
>From [Azure portal](https://portal.azure.com), go to your MAG Azure Storage account > **Containers > [mag-yyyy-mm-dd] > samples > CreateFunctions.usql > Download**. [](start = 4, length = 164)

From [Azure portal](https://portal.azure.com), go to your MAG Azure Storage account and download **/\<MAG_CONTAINER\>/\<MAG\>/\<YYYY-MM-DD\>/samples/usql/CreateFunctions.usql**.